realm/realm-jni/src/io_realm_internal_table.cpp

```diff
@@ -1551,7 +1551,7 @@ bool check_valid_primary_key_column(JNIEnv* env, Table* table, StringData column
                     int64_t next_val = results.get_int(column_index, i);
                     if (val == next_val) {
                         std::ostringstream error_msg;
-                        error_msg << "Field \"" << table->get_column_name(column_index).data() << "\" cannot be a primary key, ";
```

Comment:
@kneth you mean that it's better to just removing the `.data()`?

If so, I agree with that.